sklearn/model_selection/_validation.py

```diff
@@ -2383,6 +2383,14 @@ def validation_curve(
 
         .. versionadded:: 1.6
 
+    See Also
+    --------
+    ValidationCurveDisplay.from_estimator : Plot the validation curve
+        given an estimator, the data, and the parameter to vary.
+    ValidationCurveDisplay.from_predictions : Plot the validation curve
+        given the training and test scores.
+    ValidationCurveDisplay : Validation curve visualization.
```

Comment:
I would just directly link to `ValidationCurveDisplay.from_estimator` and not to the `ValidationCurveDisplay` class itself, as using the class constructor is not recommended to link to use the constructor.

```suggestion
```